config/prod/new-test-instance-for-minneker.yaml

```diff
@@ -0,0 +1,49 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-v9.j2
```

Comment:
Use new template

Reply:
I believe I updated it to the newest template, please let me know if not!